scripts/valhalla_build_config

```diff
@@ -288,7 +289,7 @@ help_text = {
         'user_agent': 'User-Agent http header to request single tiles',
         'tile_url': 'Http location to read tiles from if they are not found in the tile_dir, e.g.: http://your_valhalla_tile_server_host:8000/some/Optional/path/{tilePath}?some=Optional&query=params. Valhalla will look for the {tilePath} portion of the url and fill this out with a given tile path when it make a request for that tile',
         'tile_url_gz': 'Whether or not to request for compressed tiles',
-        'concurrency': 'How many threads to use in the concurrent parts of tile building',
+        'concurrency': 'How many threads to use in the concurrently running executables or parts of executables. Default all.',
```

Comment:
depending on a persons familiarity with valhalla this might be misleading. i think it only applies to executables that are for data buildings tools, at present right? i mean its inside the mjolnir part of the config but im just saying, its not ALL executables.

Reply:
`Default all` refers to all threads, not all executables